### PR TITLE
issue-22 - Links not properly styled

### PR DESCRIPTION
**Pull request by alinarublea**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--zeiss--hlxsites.hlx.page/en/semiconductor-manufacturing-technology/news-and-events/smt-press-releases/intels-preferred-quality-supplier-award-2020
- After: https://issue-22--zeiss--hlxsites.hlx.page/en/semiconductor-manufacturing-technology/news-and-events/smt-press-releases/intels-preferred-quality-supplier-award-2020

### CODE REVIEW

File: styles/simplified-styles.css [hunk 1]
```diff
@@ -46,6 +46,35 @@ a:hover {
     color: var(--button-primary-border-hover);
 }
 
+a.button.secondary {
```
Comment (jindaliiita) [inline styles/simplified-styles.css:49]: Can we somehow incorporate these styles of columns-new.css here ?
Means can we remove styling from columns-new.css and make a common styling here
https://github.com/hlxsites/zeiss/blob/main/blocks/columns-new/columns-new.css#L19-L42

Comment (alinarublea) [reply styles/simplified-styles.css:49]: @jindaliiita the simplified styles is now the same as it was in columns-new, so I removed the in columns new, however for this to work all the links in the columns new block in the docs need to be Italic (secondary button style). Is it ok to manually change them or should I update the import script

Comment (jindaliiita) [reply styles/simplified-styles.css:49]: Yes, it's fine for me but i am not sure how are we tracking the manual changes in different docs.

Comment (jindaliiita) [reply styles/simplified-styles.css:49]: Currently there is only one doc which is having link in columns-new.
https://main--zeiss--hlxsites.hlx.page/en/semiconductor-manufacturing-technology/news-and-events/smt-press-releases/intels-preferred-quality-supplier-award-2020

Comment (alinarublea) [reply styles/simplified-styles.css:49]: Let me change that one and update the test urls

Comment (jindaliiita) [reply styles/simplified-styles.css:49]: Its not reflected in the test url

Comment (alinarublea) [reply styles/simplified-styles.css:49]: Yes, it doesn't work yet, trying to figure out why the first produces a secondary button and the firrst doesn't https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7BF9591F8C-9158-4289-B2EC-AFBC72731340%7D&file=intels-2022-epic-distinguished-supplier-award-en-only.docx&action=default&mobileredirect=true&cid=880f5ff7-0d0b-40c1-bfc9-3059b48d56aa vs https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7BB9BAF4F3-8C6A-44D1-8A5A-B0B36B04C0AA%7D&file=intels-preferred-quality-supplier-award-2020.docx&action=default&mobileredirect=true&cid=83f89709-c369-4671-ab3b-89a1b4a524f3

Comment (alinarublea) [reply styles/simplified-styles.css]: @jindaliiita I changed the rule and now it works for the example page you sent

**Review by jindaliiita — APPROVED** [review]

**Pull request merged**